client/packages/system/src/Manage/Preferences/EditPage/EditPreferencesPage.tsx

```diff
@@ -26,7 +26,6 @@ export const EditPreferencesPage = () => {
           return (
             <InputWithLabelRow
               key={pref.key}
-              labelRight
```

Comment:
Removed since looks weird now with 'non-boolean' selection... switch selection is all the way to the right so looks a lot better than it did better when it was right next to the label~